typographer/backends/skia/text_render_context_skia.cc

```diff
@@ -122,22 +120,46 @@ static std::optional<SkBitmap> CreateAtlasBitmap(const GlyphAtlas& atlas,
                                const Rect& location) -> bool {
     const auto position = SkPoint::Make(location.origin.x, location.origin.y);
     SkGlyphID glyph_id = font_glyph.glyph.index;
-    SkFont font(
+
+    SkFont sk_font(
         TypefaceSkia::Cast(*font_glyph.font.GetTypeface()).GetSkiaTypeface(),
         font_glyph.font.GetMetrics().point_size);
 
-    SkFontMetrics metrics;
-    font.getMetrics(&metrics);
+    const auto& metrics = font_glyph.font.GetMetrics();
+
+    auto glyph_color = SK_ColorWHITE;
+
+#if 0
+    {
+      glyph_color = SkColorSetARGB(255,                //
+                                   std::rand() % 255,  //
+                                   std::rand() % 255,  //
+                                   std::rand() % 255   //
+      );
+      SkPaint debug_paint;
+      debug_paint.setARGB(255 / 4,            //
+                          std::rand() % 255,  //
+                          std::rand() % 255,  //
+                          std::rand() % 255   //
+      );
+      canvas->drawRect(SkRect::MakeXYWH(location.origin.x,    //
+                                        location.origin.y,    //
+                                        location.size.width,  //
+                                        location.size.height  //
+                                        ),
+                       debug_paint);
+    }
+#endif
```

Comment:
Is this for checking if things are getting mapped outside the bounding box? Seems useful to keep it around for quick checking.

Reply:
Yeah, I kept adding and backing it out. Thought I'd leave it in there for a bit.